### PR TITLE
Fix misdirected links in Web UI from pod conversion

### PR DESCRIPTION
**Pull request by bcbroussard**
From the comments in #7122 .

Fixes the blue Kubernetes logo link and a number of others.

cc /@lavalamp @bgrant0607

### CODE REVIEW

**Comment by bgrant0607** [comment]
@jackgr: please review

File: www/app/components/dashboard/views/pod.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
           <tr>
             <td class="name">Status</td>
             <td class="value">
-              {{pod.currentState.status}} on <a ng-href="/#/dashboard/groups/host/selector/host={{pod.currentState.host}}">{{pod.currentState.host}}</a>
+              {{pod.currentState.status}} on <a ng-href="#/dashboard/groups/host/selector/host={{pod.currentState.host}}">{{pod.currentState.host}}</a>
```
Comment (bgrant0607) [inline www/app/components/dashboard/views/pod.html]: Please convert to v1beta3 syntax. We're eliminating v1beta1 within the next week or two.

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api.md#v1beta3-conversion-tips
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go

File: www/master/components/dashboard/views/listPodsCards.html [hunk 2]
```diff
@@ -43,7 +43,7 @@
               <md-grid-tile-footer>
 
                 <div class="pod-host">{{pod.currentState.host}}</div>
-                <div><a ng-href="/#/dashboard/pods/{{ pod.id }}"><h3>{{ pod.id }}</h3></a></div>
+                <div><a ng-href="#/dashboard/pods/{{ pod.id }}"><h3>{{ pod.id }}</h3></a></div>
```
Comment (bgrant0607) [inline www/master/components/dashboard/views/listPodsCards.html]: id is gone. now metadata.name

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api.md#v1beta3-conversion-tips

**Comment by bcbroussard** [comment]
@bgrant0607 Thanks, I've changed to this to WIP until the v1beta3 updates are in.

**Comment by lavalamp** [comment]
LGTM, rerunning shippable. I think it's reasonable to fix @bgrant0607's comments in another PR. But please do it soon, because v1beta1 is going away.

**Comment by bgrant0607** [comment]
Does this overlap at all with #8332?

**Comment by bcbroussard** [comment]
@bgrant0607 There is no overlap with #8322. You can go ahead and merge this, and @preillyme will rebase to incorporate these changes into the UI binary.

**Comment by bcbroussard** [comment]
@lavalamp I'll be working on v1beta3 support after these UI fixes. Should be finished by early next week at the latest.

**Comment by bgrant0607** [comment]
One shippable run passed. The other looked like an infrastructure failure, not a test failure. It doesn't look like travis will run any time soon. I'm not sure we have any test coverage of this, anyway. 

LGTM.

**Comment by preillyme** [comment]
:+1:

**Comment by jackgr** [comment]
:+1:

**Pull request merged**